src/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -115,6 +115,38 @@ SentrySDK.start { options in
 [NSString]: https://developer.apple.com/documentation/foundation/nsstring
 [NSBundle]: https://developer.apple.com/documentation/foundation/nsbundle
 
+## Core Data Instrumentation
+
+<Alert level="info" title="Important">
+
+This is an experimental feature and requires an opt-in. Experimental features are still a work-in-progress and may have bugs.
+
+</Alert>
+
+The Sentry SDK adds spans for Core Data operations to ongoing transactions bound to the scope. Currently, the SDK supports fetch and save operations with [NSManagedObjectContext][NSManagedObjectContext].
```

Comment:
Are we reusing these URLs? Isn't it easier to inline them?

```suggestion
The Sentry SDK adds spans for Core Data operations to ongoing transactions bound to the scope. Currently, the SDK supports fetch and save operations with [NSManagedObjectContext](https://developer.apple.com/documentation/coredata/nsmanagedobjectcontext).
```

If not, please add it to the bottom of the page with the rest, but imo that just makes things more complicated to maintain